install/python-installer/traf_setup.py

```diff
@@ -50,6 +50,9 @@ def run():
     ### kernel settings ###
     run_cmd('sysctl -w kernel.pid_max=65535 2>&1 > /dev/null')
     run_cmd('echo "kernel.pid_max=65535" >> /etc/sysctl.conf')
+    run_cmd('cp $SQ_ROOT/sysinstall/etc/init.d/trafodion /etc/init.d')
```

Comment:
I think @mkby is right. The run_cmd will not have SQ_ROOT in the shell environment. The cp command needs to be changed as he suggests.